packages/netlify-cms-core/src/constants/configSchema.js

```diff
@@ -128,7 +136,8 @@ const getConfigSchema = () => ({
       },
     },
   },
-  required: ['backend', 'media_folder', 'collections'],
+  required: ['backend', 'collections'],
+  oneOf: [{ required: ['media_folder'] }, { required: ['media_library'] }],
```

Comment:
`oneOf: []` will prevent `media_folder` from being set if `media_library` is. If there is a possibility a media library integration would *also* want `media_folder` to be set, we should use `anyOf: []` instead. If you do want them to be mutually exclusive, we would leave it as `oneOf: []`.

It took me forever to understand the difference, so I figured I'd note it here. :smile:

Reply:
Yeah I actually meant to change that, thanks for the reminder.